src/controllers/error.js

```diff
@@ -0,0 +1,6 @@
+exports.client = (req, res) => {
+  res.render('404',{layout:false});
```

Comment:
ok its good :+1:  , but now if i im 404 page how i can return back to home or anther page ? :relaxed: 
so i think you need to add button or any thing to do its :smile:

Reply:
i did it ^_^ "